wrappers.c

```diff
@@ -23,6 +23,10 @@ extern int __unix_errno;
 #define SI86FPHW 40
 #define FP_387 3
 
+#define TCGETS 0x5401
```

Comment:
Hmm, maybe we're just missing an include?

Reply:
No, I don't believe that BSD defines these `ioctl(2)` magic numbers. They are likely included on Linux for compatibility with legacy software, but POSIX says we should use `tcgetattr(fd, argp)` instead.